### PR TITLE
Added a documentation of a opsdroid skill

### PR DESCRIPTION
**Pull request by imrishabh18**
# Description

Added a documentation of a opsdroid skill.

Fixes #<1198>

## Status
UNDER DEVELOPMENT

## Type of change

- This change requires a documentation update
- Documentation (fix or adds documentation

# How Has This Been Tested?

# Checklist:

- [ ✔] I have performed a self-review of my own code
- [ ✔ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/examples/random.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# opsdroid skill-random
```
Comment (jacobtomlinson) [inline docs/examples/random.md]: Let's have a more useful title here that describes what this page is about. Perhaps something like 'Creating a skill for making random chance decisions'.

File: docs/examples/random.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+A skill for [opsdroid](https://github.com/opsdroid/opsdroid) to perform random events like flipping a coin or rolling a dice.
```
Comment (jacobtomlinson) [inline docs/examples/random.md]: We are talking to the reader here and telling them what we will cover in this page. Perhaps something like 'we will create a skill for performing random events...'

You can go into more detail about what the skill will do, why is it useful?

File: docs/examples/random.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+## Creating skills
+
+Skills are designed to be simpler than other modules to ensure that it is easy to get started.
+
+To create a skill you need to at minimum create a single python file in your repository with the  `__init__.py`  name. For example the skill  `hello`  has a single file called  `__init__.py`.
```
Comment (jacobtomlinson) [inline docs/examples/random.md]: We don't care about generic information on 'creating skills', that already exists in the documentation, this is a tutorial on creating a specific skill. 

This section should be talking the user through how to create the right folder and files for the random chance skill they are creating.

File: docs/examples/random.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+##  Configuration
+```yaml
+None.
+```
```
Comment (jacobtomlinson) [inline docs/examples/random.md]: This section should not say `None.`.

How does the user configure opsdroid to use their random chance skill? Tell them that here.

File: docs/examples/random.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+@match_regex(r'flip a coin', case_sensitive=False)
+
+async def flip_a_coin(opsdroid, config, message):
+
+	result = random.choice(["heads", "tails"])
+
+	text = random.choice(COIN_FLIP_RESPONSES).format(result=result)
+
+	await message.respond(text)
+```
```
Comment (jacobtomlinson) [inline docs/examples/random.md]: Copying and pasting the skill code here is not good enough. There aren't even any comments in the code explaining what is going on.

You should slowly walk the user through each section, explaining why it is there and what it does. What does each function call do, how does it help?

You should definitely include the full skill code in a single block like this, just after you have explained it all in pieces.

File: docs/examples/random.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+> user: flip a coin
+>opsdroid: It landed on heads
+
+    roll a dice
+   Rolls a dice.
+   
+
+> user: roll a dice
+>opsdroid: You rolled a 4
```
Comment (jacobtomlinson) [inline docs/examples/random.md]: The markdown formatting is off here. You could add more words to explain to the user what is happening here.

File: mkdocs.yml [hunk 7]
```diff
@@ -13,6 +13,7 @@ pages:
       - "Example Skills":
               - "Introduction": "examples/introduction.md"
               - "Weather": "examples/weather.md"
+              - "random": "examples/random.md"
```
Comment (jacobtomlinson) [inline mkdocs.yml]: This is the documentation index. Please use correct capitalisation and try to make it clear what the skill does in a short number of words. Perhaps like this.

```suggestion
              - "Random Chance": "examples/random.md"
```

**Comment by imrishabh18** [comment]
@jacobtomlinson Thank you for your review. I am working on the changes.

**Pull request closed without merging**